docker/AccountManager.Dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1
+FROM python:3.9.9-buster
+
+WORKDIR /usr/local/share/
+RUN git clone https://github.com/DarkflameUniverse/AccountManager
+
+WORKDIR AccountManager
+ADD docker/credentials_example.py credentials.py
+ADD docker/resources_example.py resources.py
+RUN pip3 install -r requirements.txt && echo "SECRET_KEY = r'$(openssl rand -base64 30)'" >> credentials.py
+
+EXPOSE 5000
+HEALTHCHECK --interval=2m --timeout=3s \
```

Comment:
Might be better to do healthchecks in the docker-compose file?

Reply:
Yes, that might be better. So the user can change intervals and such.